### ISSUE TITLE
quiet in pyproject.toml not respected when isort run from code

### ISSUE THREAD

**Issue by czerus** [PyCQA/isort#1461]
Hi

I have an isort settings in pyproject.toml that looks like follow:

```
[tool.isort]
sections = "FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER"
line_length = 120
known_first_party = "libName"
case_sensitive = true
quiet = true
```

and using isort in python script (together with autopep) to format the code: `isort.file(path_to_file)`. When isort finds that something needs to be reformatted in a file it prints: "Fixing path_to_file". So it does not respect the configuration from toml. In order to fix that I need to pass the custom `Config` instance to `file` method but this overwrites all the settings from toml. Wha is strange is that isort run from command line properly handles this quiet in toml and does not prints "Fixing...."

I am using python 3.6.8 and isort 5.5.0

**Comment by timothycrosley** [comment]
Hi @czerus,

Thank you for the report, and I am sorry you are encountering this issue!

My initial hunch is that the most likely thing is that when you call it from the API it's not finding your configuration file. You can manually specify the file or path:

```python
isort.sort_file("my_file.py", settings_file="full/path/to/pyproject.toml")
```

Additionally, you can always override single settings just by passing them in, similar to how the CLI allows overriding similar settings:

```python
isort.sort_file("my_file.py", quiet=True)
```

Let me know if these suggestions help resolve your issue!

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
Also, if you are doing this on more than one file in your script, I recommend reusing the configuration to help improve performance:

```
import isort

isort_config = isort.Config(settings_file=".isort.cfg", quiet=True)

for file in files:
    isort.file(file, config=isort_config)
```

**Comment by czerus** [comment]
Thank you for such fast reply 👍 
I have used 

```
isort.file(file, settings_file="tests/pyproject.toml")
```
and made sure that in debugger quiet is set to `True` (and any other option set in toml is visible) but the "Fixing ..." keeps appearing. So I tried this one:
```
isort_config = isort.Config(settings_file="tests/pyproject.toml")
```
and debugger shows that `config.quiet` is True. But when I have set the same path in `isort.file()` then quiet (in api.py) is False.

What works is providing the whole config that is earlier read from toml:
```
isort_config = isort.Config(settings_file="tests/pyproject.toml")
is_fixed_isort = isort.file(file, config=isort_config)
```

**Comment by timothycrosley** [comment]
@czerus Thanks for all the information! I was able to isolate the cause of quiet not being skipped unless the config was predefined, and I’ve fixed it in develop. This will go out with the 5.6.0 release of isort.

Thanks!

~Timothy

**Comment by czerus** [comment]
Super! Glad I could help. @timothycrosley  when can we expect this 5.6.0 release?

**Comment by timothycrosley** [comment]
The expected release date is on or before October 1st

**Comment by timothycrosley** [comment]
Update: this has just been released to PyPI in 5.6.0 release of isort: https://pycqa.github.io/isort/CHANGELOG/#560-october-7-2020

Thanks!

~Timothy
